packages/orbit-components/.storybook/main.ts

```diff
@@ -61,10 +61,6 @@ const config: StorybookConfig = {
     }
     return undefined;
   },
-
-  typescript: {
-    reactDocgen: "react-docgen-typescript",
-  },
```

Comment:
Curiosity, was this the root cause or the change in path patterns?

Reply:
It was this one, yes. The path would just make storybook look for mdx files, which we don't have, so I also removed it.